runtime/j9vm/j9vmnatives.xml

```diff
@@ -333,9 +333,6 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JVM_GetNestMembers"/>
 		<export name="JVM_AreNestMates"/>
 		<export name="JVM_InitClassName"/>
-	</exports>
-
-	<exports group="jdk12">
```

Comment:
The group should stay; it's referenced in `module.xml`.

Reply:
Or be deleted from there as well since there are no exports in it.

Reply:
Running some sanity tests now.

Reply:
Update: removed empty group `jdk12`, verified `JDK11/12` builds.
@DanHeidinga please have another look.